src/Generics/SOP/GGP.hs

```diff
@@ -294,7 +294,7 @@ gfrom x = gSumFrom (GHC.from x) (error "gfrom: internal error" :: SOP.SOP SOP.I
 -- For more info, see 'Generics.SOP.Generic'.
 --
 gto :: forall a. (GTo a, GHC.Generic a) => SOP I (GCode a) -> a
-gto x = GHC.to (gSumTo x id ((\ _ -> error "inaccessible") :: SOP I '[] -> (GHC.Rep a) x))
+gto x = GHC.to (gSumTo x id (\case :: SOP I '[] -> (GHC.Rep a) x))
```

Comment:
I'm not a fan of lambda case, and would prefer this to be expanded. But I can also do it myself after merging.